backend/geonature/core/gn_meta/routes.py

```diff
@@ -772,11 +772,17 @@ def delete_acquisition_framework(scope, af_id):
         raise Forbidden(
             f"User {g.current_user} cannot delete acquisition framework {af.id_acquisition_framework}"
         )
-    if not af.is_deletable():
+    if af.has_datasets():
         raise Conflict(
             "La suppression du cadre d’acquisition n'est pas possible "
             "car celui-ci contient des jeux de données."
         )
+
+    if af.has_child_acquisition_framework():
+        raise Conflict(
+            "La suppression du cadre d’acquisition n'est pas possible "
+            "car celui-ci est le parent d'autre(s) cadre(s) d'acquisition."
```

Comment:
```suggestion
            "La suppression du cadre d’acquisition est impossible "
            "car celui-ci est le parent d'autre(s) cadre(s) d'acquisition."
```

Reply:
improve "is parent of other" error message